Transient memory leak in Windows test in `VectorSortTest`

```
[error] Test org.enso.interpreter.test.VectorSortTest failed: java.lang.AssertionError: EnsoContext can be GCed when context is closed ref still alive: org.enso.interpreter.runtime.EnsoContext@73512b31
```

heapdump is then generated, but it is damaged and cannot be opened. Probably because line endings?
`Loading failed: Heap dump is broken. Tag 0x2c is missing`

Occurences:
- https://github.com/enso-org/enso/actions/runs/19178030016/job/54831920902?pr=14268#step:14:6208
- https://github.com/enso-org/enso/actions/runs/19143033879/job/54822503836?pr=14252#step:14:6089
- https://github.com/enso-org/enso/actions/runs/19264563993/job/55093758462?pr=14268#step:14:6898

--- a/engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/VectorSortTest.java
+++ b/engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/VectorSortTest.java
@@ -22,7 +22,9 @@ import org.junit.runner.RunWith;
  */
 @RunWith(Theories.class)
 public class VectorSortTest {
-  @ClassRule public static final ContextUtils ctxRule = ContextUtils.createDefault();
+  @ClassRule
+  public static final ContextUtils ctxRule = ContextUtils.newBuilder().assertGC(false).build();
+
   private static Value sortFunc;
   private static Value equalsFunc;
 

